source/crud/tests/unified/find.yml

```diff
@@ -0,0 +1,63 @@
+description: "find"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: true # ensure cursors pin to a single server
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name find-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+      - { _id: 4, x: 44 }
+
+tests:
+  - description: "find with multiple batches works"
+    operations:
+      - name: find
+        arguments:
+          filter: { _id: { $gt: 1 } }
+          batchSize: 1
```

Comment:
Quoting [`find` command docs](https://docs.mongodb.com/manual/reference/command/find/):

> Unlike the previous wire protocol version, a batchSize of 1 for the find command does not close the cursor.

I wasn't aware that the `find` command differed in this respect from OP_QUERY, but given that this test file doesn't specify a `minServerVersion` I think we should continue to use `batchSize: 2` for compatibility unless you'd like to specify `minServerVersion: "3.2" for these tests.

Reply:
Updated to use batchSize 2